yaml.cabal

```diff
@@ -1,5 +1,5 @@
 name:            yaml
-version:         0.8.17.2
+version:         0.8.17.3
```

Comment:
This is a patch bump. Since this is a potentially surprising change in behavior, I'd rather a minor version bump (which can be tracked via Cabal CPP macros). In this case: 0.8.18.